packages/lesswrong/server/notificationTypesServer.tsx

```diff
@@ -501,3 +502,33 @@ export const PostCoauthorAcceptNotification = serverRegisterNotificationType({
     );
   },
 });
+
+export const NewSubforumMemberNotification = serverRegisterNotificationType({
+  name: "newSubforumMember",
+  canCombineEmails: false,
+  emailSubject: async ({ user, notifications }: {user: DbUser, notifications: DbNotification[]}) => {
+    const newUser = await Users.findOne(notifications[0].documentId)
+    if (!newUser) throw new Error("Cannot find user for which this notification is being sent")
+    return `New member ${newUser.displayName} has joined your subforum`;
+  },
+  emailBody: async ({ user, notifications }: {user: DbUser, notifications: DbNotification[]}) => {
+    const newUser = await Users.findOne(notifications[0].documentId)
+    const subforum = await Tags.findOne(notifications[0].extraData?.subforumId)
+
+    return <div>
+      <p>
+        Hi {user.displayName},
+      </p>
+      <p>
+        Your subforum, <a
+          href={combineUrls(getSiteUrl(), `/${taggingNameIsSet.get() ? taggingNamePluralSetting.get() : "tag"}/${subforum?.slug}/subforum`)}
```

Comment:
You could use `tagGetSubforumUrl` here

Reply:
Nice, thanks. That needed to have been written.